Src/Microsoft.Dynamic/ComInterop/ComTypeDesc.cs

```diff
@@ -5,24 +5,18 @@
 #if FEATURE_COM
 
 using System;
-using System.Collections;
 using System.Collections.Generic;
+using System.Globalization;
 using System.Runtime.InteropServices.ComTypes;
 using System.Threading;
 
 namespace Microsoft.Scripting.ComInterop {
 
     public class ComTypeDesc : ComTypeLibMemberDesc {
-        private string _typeName;
-        private string _documentation;
-        //Hashtable is threadsafe for multiple readers single writer. 
```

Comment:
I have not looked at usage. But I assume this comment is here because it's intended to be used in multithreaded scenarios? Dictionary doesn't provide the same thread safety guarantees as Hashtable.

Reply:
I used ConcurrentDictionaries instead. What do you think?

Reply:
Using ConcurrentDictionary seems fine from a thread safety point of view. Hopefully it isn't a perf hit. Just need to find time to look at how these classes are used and will merge it in assuming there are no issues.